scripts/common_startup.sh

```diff
@@ -9,6 +9,7 @@ done
 # Conda Python is in use, do not use virtualenv
 if python -V 2>&1 | grep -q -e 'Anaconda' -e 'Continuum Analytics' ; then
     SET_VENV=0
+    CONDA_ALREADY_INSTALLED=1
```

Comment:
Seems reasonable to me to echo the statement at the point of the logical difference, so, here.